addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```diff
@@ -0,0 +1,81 @@
+jwt.settings.title=JWT Settings
+jwt.toolmenu.settings=JWT Settings...
+ascanrules.jwt.name=JWT Scanner 
+ascanrules.jwt.description=Scanner for finding vulnerabilities in JWT implementations.
+jwt.settings.rsa.header=<html><h3>RSA:<h3></html>
+jwt.settings.general.header=<html><h3>General:<h3></html>
+jwt.settings.rsa.trustStorePath=TrustStore Path:
+jwt.settings.rsa.trustStorePassword=TrustStore Password:
+jwt.settings.rsa.trustStoreFileDescription=PKCS12 format
+jwt.settings.filechooser.button=Select...
+jwt.settings.general.ignoreClientSideScan.checkBox=Ignore Client Configuration Scan
```

Comment:
Is "ignore" the right word here? Is it actually used an then ignored or would "Disable" be more correct?

Further if "Disable" is more correct should the logic be reversed? ex: Default the state to checked and make the label "Enable Client Configuration Scan"?

Reply:
hmm... i think enable is a better option. will do the changes.